dashboard/lib/tasks/seed.rake

```diff
@@ -102,6 +102,8 @@ namespace :seed do
     coursec-2019
     coursee-2019
     coursea-2020
+    csd3-2023
+    interactive-games-animations-2023
```

Comment:
as of this PR we are reading the contents of the rubrics from the database for the units in the UNIT_AND_LEVEL_TO_LESSON_S3_NAME map. in order for these rubrics to be present in the database during UI tests, those units must be in this list.